### PR TITLE
Remove/replace newline chars in fieldComments

### PR DESCRIPTION
**Pull request by janeymunoz**
Addresses #8

### CODE REVIEW

File: src/Database/Persist/Documentation.hs [hunk 1]
```diff
@@ -297,7 +297,7 @@ markdownTableRenderer = Renderer{..}
         , "` | "
         , showType fieldSqlType
         , " | "
-        , fold mextra
+        , fold $ fmap cleanComment mextra
```
Comment (parsonsmatt) [inline src/Database/Persist/Documentation.hs]: i think `fold . fmap f` is `foldMap f`

File: src/Database/Persist/Documentation.hs [hunk 2]
```diff
@@ -344,6 +344,19 @@ markdownTableRenderer = Renderer{..}
...
+     let
+       -- Newline characters may be present in a comment. The need to be removed
+       -- to allow the markdown tables to render properly.
+       newlineToBr '\n' = "<br>"
+       newlineToBr c = Text.singleton c
+     in
+      case Text.unsnoc comment of
+        Nothing -> comment
+        Just (front, '\n') -> Text.concatMap newlineToBr front
+        Just _ -> Text.concatMap newlineToBr comment
```
Comment (parsonsmatt) [inline src/Database/Persist/Documentation.hs]: May be worthwhile to use [`Text.strip`](https://www.stackage.org/haddock/lts-18.3/text-1.2.4.1/Data-Text.html#v:strip). Leading and/or trailing whitespace won't get rendered anyway, and it should clean out the implementation nicely.

**Review by parsonsmatt — APPROVED** [review]

File: src/Database/Persist/Documentation.hs (hunk 2 above)
Comment (janeymunoz) [reply src/Database/Persist/Documentation.hs]: done!

**Comment by parsonsmatt** [comment]
uploaded 0.1.0.4

**Pull request merged**